docs/reindexing-content.md

```diff
@@ -6,24 +6,26 @@ When making changes the data or analyzers that are stored in the search index, y
 
 The `elasticpress` CLI command is used for all index management operations.
 
-To re-sychronize the index for all sites
+As these tasks are long-running commands, it is recommended to run the index on individual sites instead of using the `--network-wide` flag. When using the `--network-wide` flag on sufficiently large networks, the CLI process would run out of memory and be killed by the operating system. To better manage memory, it is recommended to use an `xargs` pipeline to process each site. To re-sychronize the index for all sites:
 
 ```sh
-wp elasticpress index --network-wide
+wp site list --field=url | xargs -I % wp elasticpress index --url=%
 ```
 
-Re-sychronise the index and updating mappings (needed when updating synonym definitions)
+`wp site list --field=url` will print each site URL on a new line and ingested by `xargs` to be processed individually.
+
+To re-sychronise the index and update mappings (needed when updating synonym definitions):
 
 ```sh
-wp elasticpress index --setup
+wp site list --field=url | xargs -I % wp elasticpress index --url=% --setup
```

Comment:
```suggestion
wp site list --field=url | xargs -I % wp elasticpress index --url=% --setup && wp elasticpress recreate-network-alias
```

Reply:
Does this need to be run once or for each network? If only once, your recommendation is sound! Bash picks up the `&&` before `xargs` does, so it's treated as a separate command from `xargs`.

Reply:
Ah that's what I was worried about, needs to be scoped to the xargs command

Reply:
Looking at the source, it appears to loop through all sites, so I believe this should run outside of the xargs command (how this PR has it currently written). What do you think?

2.8.2: https://github.com/10up/ElasticPress/blob/b56e8f8dc2f07cd6c591edc5db94eb49236bc549/bin/wp-cli.php#L282-L310

latest: https://github.com/10up/ElasticPress/blob/c8c686d49f588d8ccfdc4e1cf0b22d9459f8276e/includes/classes/Command.php#L456-L486

Reply:
Missed that bit! Alrighty then, we're good here